ovs/v1/ovs.yaml

```diff
@@ -436,15 +471,15 @@ paths:
                   required: true
                   explode: true
                   schema:
-                    $ref: '#/components/headers/signatureHeader'
+                    $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_OVS_DOMAIN/1.0.0#/components/headers/signatureHeader'
```

Comment:
I think you should use the 'signatureHeaderParam' defined in DCSA_NOTIFICATION_DOMAIN (take a look in EBL) - this way we use the same signature all places